sagemaker-pipelines/step-decorator/local-mode-example/using_step_decorator_with_local_mode.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        booster = xgboost.train(
</pre><p> </p><p>This would work well but can we change to use XGBRegressor instead? The rationales are:</p><ol><li>The XGBRegressor supports more types of data input e.g. numpy, pandas DF</li><li>Once this PR is merged, I plan to do some exploration and extend it to include the ModelBuilder local mode introduced by Galactus, so that it can be a complete ML story to end up with model inference. However, <strong>ModelBuilder does not support the Booster type well</strong>. Thus it would be easier if we switch to use the more powerful XGBRegressor</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>